test/TEST-20-MAINPIDGAMES/testsuite.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+set -ex
+set -o pipefail
+
+systemd-analyze set-log-level debug
+systemd-analyze set-log-target console
+
+test `systemctl show -p MainPID --value testsuite.service` -eq $$
```

Comment:
Ok, I lied, this won't work on RHEL 7 as well, as `--value` is not present there.

Reply:
(The same applies to all `systemctl --value` calls below)

Reply:
Fixed.